app/views/accounts/actions/_manage_backup_codes.html.slim

```diff
@@ -0,0 +1,4 @@
+= link_to 'backup_code_setup' do
```

Comment:
How about a path helper instead of a hardcoded path, eg. `backup_code_setup_url` or `backup_code_setup_path`?